WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -92,6 +96,7 @@ public void onCreate(Bundle savedInstanceState) {
         restoreSavedInstanceState(savedInstanceState);
         setupActionBar();
         setupRecycleView();
+        initSwipeToRefreshHelper(findViewById(android.R.id.content));
```

Comment:
Most definitely will fix this. My refreshes finish almost instantly so I never saw this happen :/